vncviewer/OptionsDialog.cxx

```diff
@@ -654,6 +684,15 @@ void OptionsDialog::createSecurityPage(int tx, int ty, int tw, int th)
     encX509Checkbox->callback(handleX509, this);
     ty += CHECK_HEIGHT + TIGHT_MARGIN;
 
+#ifdef HAVE_NETTLE
+    encRSAAESCheckbox = new Fl_Check_Button(LBLRIGHT(tx, ty,
+                                                   CHECK_MIN_WIDTH,
+                                                   CHECK_HEIGHT,
+                                                   _("RSA-AES")));
+    encRSAAESCheckbox->callback(handleRSAAES, this);
+    ty += CHECK_HEIGHT + TIGHT_MARGIN;
+#endif
```

Comment:
The height of the containing `Fl_Group` needs to be adjusted as well. It should also be dynamic based on `#ifdef`s.